react/components/EXPERIMENTAL_Select/index.js

```diff
@@ -16,140 +18,159 @@ const getOptionValue = option => {
   return JSON.stringify(option.value)
 }
 
-const Select = ({
-  autoFocus,
-  creatable,
-  defaultValue,
-  disabled,
-  errorMessage,
-  label,
-  loading,
-  multi,
-  noOptionsMessage,
-  onChange,
-  onSearchInputChange,
-  options,
-  placeholder,
-  size,
-  value,
-}) => {
-  const reactSelectComponentProps = {
-    autoFocus,
-    className: `pointer b--danger bw1 ${getFontClassNameFromSize(size)} ${
-      errorMessage ? 'b--danger bw1' : ''
-    }`,
-    components: {
-      ClearIndicator,
-      Control: function Control(props) {
-        return (
-          <ControlComponent
-            errorMessage={errorMessage}
-            size={size}
-            {...props}
-          />
-        )
-      },
-      DropdownIndicator: function DropdownIndicator(props) {
-        return <DropdownIndicatorComponent size={size} {...props} />
+class Select extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
```

Comment:
If you are not gonna manipulate (setState) then this should be just a const. it doesn't have to be a state.
```suggestion
    this.inputId = `react-select-input-${uuid()}`
```

Reply:
Or maybe it could be a `static` inside the class.